src/reducers/index.js

```diff
@@ -10,11 +10,22 @@ const initialState = {
 // by our action creator. What does the reducer
 // need to do with the count in each case?
 export default (state = initialState, action) => {
+  console.log('reducer running', action)
   switch (action.type) {
     case INCREMENT:
+      return {count: state.count +1};
+
     // Fill in the body of this case
     case DECREMENT:
+      return Object.assign({}, state, {count: state.count-1});
     // Fill in the body of this case
+    
+    case INCREMENTIFODD:
+      return Object.assign({}, state, {count: state.count+2});
```

Comment:
This is where you will want to do some conditional logic. You could use ternary syntax to check the value of state.count and increment the count only if it is odd